Sources/App/Views/PublicPage.swift

```diff
@@ -111,9 +111,9 @@ class PublicPage {
             .div(
                 .class("inner"),
                 .a(
-                    .href("/"),
+                    .href(Root.home.absolutePath),
                     .h1(
-                        .img(.src("/images/logo.svg")),
+                        .img(.src(Root.images("logo.svg").absolutePath)),
```

Comment:
This is an example of how we can transition other stringly static paths into the scheme, for instance JS & CSS paths.

I'm not 100% in love with the name `Root` for the top level resource but I can't think of anything better right now.

Reply:
Maybe `SiteURL`?